CesiumGltf/src/FeatureIdTextureView.cpp

```diff
@@ -35,6 +39,13 @@ FeatureIdTextureView::FeatureIdTextureView(
     return;
   }
 
+  if (static_cast<size_t>(texture.sampler) >= model.samplers.size()) {
```

Comment:
Just for extra safety:

```suggestion
  if (texture.sampler < 0 || static_cast<size_t>(texture.sampler) >= model.samplers.size()) {
```